presto-hive/src/main/java/com/facebook/presto/hive/metastore/ExtendedHiveMetastore.java

```diff
@@ -94,13 +94,9 @@
 
     Set<RoleGrant> listRoleGrants(PrestoPrincipal principal);
 
-    Set<String> getRoles(String user);
```

Comment:
Extract separate commit which just removes `getRoles` and replaces it's usages with `listRoleGrants`